src/components/shopModal.html

```diff
@@ -23,7 +23,7 @@ <h4 class="col-6 col-sm-4 col-lg-2">
                                      data-bind="template: { name: 'caughtStatusTemplate', data: {'status': $data.getCaughtStatus()}}">
                                </span>
                            </knockout>
-                           <img src="" height="36px" data-bind="attr:{ src: 'assets/images/items/' + name().replace(/\W/g, '_') + '.png' }">
+                           <img src="" height="36px" data-bind="attr:{ src: 'assets/images/items/' + name().replace(/[. ]/g, '_') + '.png' }">
```

Comment:
I'm not sure about this change as it will allow disallowed filenames to be tried still:
`Unown (?).png`
I think we would be better off maybe using allowed chars instead of a list of disallowed characters.
Maybe like:
```
/[^\w.-]/g
```
Which would allow any A-Z chars, 0-9 digits, `_`, `.` and `-` in the filename.
Or `/[^\w.-\\(\\)]/g` if we also want to allow `()`.

Reply:
The other files have a `_` instead spaces, so we need to standardize the file names. Like the `Unown (?).png` the burmy have spaces instead `_`, so if we allow it, we dont need to change to `_`,  but Mr. Mime file is `Mr__Mime.png` cause of dot and space, if we apply the `/[^\w.-\\(\\)]/g` the file need to be `Mr._Mime.png`.

Reply:
I think the `.` looks okay in the `Mr._Mime.png` filename, we could dissallow dots aswell though if we want?
`/[^\w-\\(\\)]/g`

Reply:
Which will basically keep all the file names the same (expect burmy)

Reply:
Ok, just spaces will be replaced so

Reply:
So it was like this:
regex: `/[^\w.-\\(\\)]/g`
Mr. Mime: `Mr._Mime.png`
Burmy (plant): Burmy_(plant)
OK?

Reply:
Sounds good 👍